l10n_br_sale_stock/tests/test_sale_stock.py

```diff
@@ -469,3 +460,43 @@ def test_button_create_bill_in_view(self):
             "Field to make invisible the Button Create Bill should be"
             " False when the Sale Order has Service and Product.",
         )
+
+    def test_compatible_with_international_case(self):
+        """Test of compatible with international case or without Fiscal Operation,
+        create Invoice but not for Brazil."""
```

Comment:
English: "Test compatibility with international cases or without Fiscal Operation,
        create an Invoice but not for Brazil."